src/main/java/com/example/staffmanagerapi/service/DocumentService.java

```diff
@@ -87,6 +88,24 @@ public Integer uploadFile(CreateDocumentDto dto, User user) throws FileEmptyExce
         return documentRepository.save(doc).getId();
     }
 
+    public byte[] downloadFile(String documentName) {
+        if (amazonS3Service.bucketIsEmpty()) {
+            throw new BadRequestException("la bucket n'existe pas ou est vide");
+        }
+        try {
+
+            final S3Object s3Object = amazonS3Service.download(BUCKET_NAME_JUSTIFICATIF, documentName);
```

Comment:
Ici `bucketIsEmpty()` peut etre `false `, et quand meme on trouve un `s3Object `qui est `null`

Reply:
si bucket is empty ça te génère direct un bad request ça ne continue même pas